### PR TITLE
Introducing Sentry Logger 🚀 

### PR DESCRIPTION
**Pull request by Rohithgilla12**
Lets go

This is a very initial basic version of Sentry integration.

As we get more issues we could add more features if needed.

### CODE REVIEW

File: packages/sentry_crispin_transport/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  sdk: ">=2.17.1 <3.0.0"
+
+dev_dependencies:
+  coverage: ^1.1.0
+  mocktail: ^0.2.0
+  test: ^1.19.2
+  very_good_analysis: ^2.4.0
+dependencies: 
+  crispin: 
+    path: ../crispin
```
Comment (johntimothybailey) [inline packages/sentry_crispin_transport/pubspec.yaml]: @Rohithgilla12 does this work in publishing? If so amazing I didn't realize you could do that

**Review by johntimothybailey — COMMENTED** [review]
First 🎉 🕺  Thank you!

Second
1. I think we should have more in the contribution file and a PR template because...
2. Can you add a similar https://github.com/DelveFore/crispin/blob/master/docs/transport-logger.mdx and link it in the Official Transports?
3. I had one clarifying question

File: packages/sentry_crispin_transport/pubspec.yaml (hunk 1 above)
Comment (Rohithgilla12) [reply packages/sentry_crispin_transport/pubspec.yaml]: Interesting question 
Actually it won't work
Good catch, sorry

References

https://github.com/felangel/bloc/blob/0b99ae0c8a156ca3d6f926cdff577a6ee7ff0768/packages/flutter_bloc/pubspec.yaml#L15

https://github.com/firebase/flutterfire/blob/c2771a425bd7260b11970e9e9e77ef40a39f9f16/packages/firebase_auth/firebase_auth/pubspec.yaml#L19

Will update them and add docs 😄

**Comment by Rohithgilla12** [comment]
Updated documentation can be found here!
https://docs.page/DelveFore/crispin~14/transport-sentry


P.S: Please have a look into this github action please, https://use.docs.page/github-bot

**Review by johntimothybailey — APPROVED** [review]

**Pull request merged**